app/controllers/idv/ssn_controller.rb

```diff
@@ -30,8 +30,9 @@ def show
 
       Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
         call('ssn', :view, true)
+      @threatmetrix_view_variables = threatmetrix_view_variables(ssn_presenter.updating_ssn?)
 
-      render 'idv/shared/ssn', locals: threatmetrix_view_variables(ssn_presenter.updating_ssn?)
```

Comment:
I find the `locals:` helpful, does it need to be pulled up into an instance variable?

Reply:
yes because I moved the locals into the actual partial that its used in.

Reply:
Is it possible to pass them through `locals:`?

```ruby
  # controller.rb
  render 'idv/shared/ssn', locals: threatmetrix_view_variables(ssn_presenter.updating_ssn?)
```

```erb
<%# my_view.html.erb %>
<%= render partial: 'shared/threat_metrix_profiling', locals: { variable1:, variable2: } %>



<%# shared/threat_metrix_profiling.html.erb %>
<%#
locals:
* threatmetrix_session_id: Session id for users threatmetrix session
* threatmetrix_javascript_urls: Url for threatmetrix javascript
* threatmetrix_iframe_url: Iframe as fallback in case javascript fails. 
%>
<% if threatmetrix_session_id.present? %>
<%# etc. %>
```